src/components/HeaderBar/index.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import styled from 'styled-components';
+import Avatar from './Avatar';
+import MenuButton from './MenuButton';
+import SwitchButton from '../SwitchButton';
+import SearchInput from './SearchInput';
+
+const Nav = styled.nav`
```

Comment:
It's a good practice to keep styled-components separated from the react components